src/nft/components/bag/BagFooter.tsx

```diff
@@ -221,12 +236,16 @@ export const BagFooter = ({
                 <Trans>Total</Trans>
               </ThemedText.SubHeaderSmall>
               <ThemedText.HeadlineSmall>
-                {formatWeiToDecimal(totalEthPrice.toString())}&nbsp;{activeCurrency?.symbol ?? 'ETH'}
+                {inputCurrency
+                  ? swapState !== TradeState.VALID
+                    ? '-'
```

Comment:
should we disable buy button when swap state is invalid?

Reply:
not right now because I am going to setup some disabling logic when I integrate with permit2 which is in the next pr

Reply:
also this isn't even the right state checking logic - I need to talk to design about some stuff so I am waiting on implementing it yet also